### ISSUE TITLE
BatchNormal

### ISSUE THREAD

**Issue by Znlsoc** [Znlsoc/LoggingForStudy#140]
>Batch Normalization 学习笔记 https://blog.csdn.net/hjimce/article/details/50866313
